doc/src/manual/conversion-and-promotion.md

```diff
@@ -98,8 +97,9 @@ function must be used to perform this operation, making it more explicit.
 To define a new conversion, simply provide a new method for `convert()`. That's really all there
 is to it. For example, the method to convert a real number to a boolean is this:
 
-```julia
-convert(::Type{Bool}, x::Real) = x==0 ? false : x==1 ? true : throw(InexactError())
+```jldoctest bool
+julia> convert(::Type{Bool}, x::Real) = x==0 ? false : x==1 ? true : throw(InexactError())
+convert (generic function with 1 method)
```

Comment:
Perhaps a better option is to rework this example entirely, with a new type, so that we can show how people actually do this in practice?